tests/unit/mainwindow/test_prompt.py

```diff
@@ -94,6 +97,31 @@ def test_backspacing_path(self, qtbot, tmp_path, get_prompt):
         prompt.item_focus('next')
         assert prompt._lineedit.text() == str(testdir / 'bar')
 
+    @pytest.mark.parametrize("keys,expected",
+                            [([], ['bar', 'bat', 'foo']),
+                            ([Qt.Key_F], ['foo']),
+                            ([Qt.Key_A], ['bar', 'bat'])])
```

Comment:
```suggestion
    @pytest.mark.parametrize("keys, expected", [
        ([], ['bar', 'bat', 'foo']),
        ([Qt.Key_F], ['foo']),
        ([Qt.Key_A], ['bar', 'bat']),
    ])
```

for readability and consistence with other tests.